README.md

```diff
@@ -22,11 +22,12 @@ Otherwise, the tool will download all `ocrd-tesserocr-recognize` models as well
 
 ## Usage
 
-- clone this repository
+- clone this repository and switch to the cloned directory
 - (optional) [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
-- run `docker compose run --build benchmarks`
-- run `bash prepare.sh`
-- run `bash workflows/execute-workflows.sh`
+- build the image with `docker compose build`
+- spin up a container with `docker compose run -d app`
+- run `docker compose exec bash prepare.sh`
+- run `docker compose exec bash workflows/execute-workflows.sh`
```

Comment:
```suggestion
- run `docker compose exec app bash workflows/execute_workflows.sh`
```

Reply:
What's `app` in this context?

Reply:
I would suggest renaming `app` to at least `quiver_app` or something. Makes easier to follow and identify directly when there are already many containers running on the host.

In general, always use very specific names to avoid potential guess work for others. I remember my first tries to install ocrd_all locally and having troubles due to collision in package versions since I already had `venv` created on my local.

Reply:
I see. Yes!